btcec/ciphering_test.go

```diff
@@ -0,0 +1,172 @@
+package btcec_test
```

Comment:
We've recently relicensed everything "The btcsuite authors".  Please update the license accordingly.